core/app/modules/kit-library/assets/js/components/taxonomies-filter.js

```diff
@@ -28,6 +28,43 @@ export default function TaxonomiesFilter( props ) {
 						taxonomiesByType={ group }
 						selected={ props.selected }
 						onSelect={ props.onSelect }
+						onOpen={ ( value, title ) => {
+							// TODO: Don't shoot this event on first load
+							if ( typeof ( value ) !== 'undefined' ) {
+								$e.run(
+									`kit-library/${ ( value ) ? 'expand' : 'collapse' }`,
+									{
```

Comment:
Please use full commands. 
Prefer
```suggestion
								const command = value ? 'kit-library/expand' : 'kit-library/collapse';
								$e.run(
									command,
									{
```